From e9aa4bbdb6dc494feca57d3985a558b37045c9a5 Mon Sep 17 00:00:00 2001
From: Rob Clark <rclark@redhat.com>
Date: Wed, 17 Dec 2014 21:38:22 -0500
Subject: [drm] radeon: re-enable dpm by default on BTC

Message-id: <1418852380-13061-12-git-send-email-rclark@redhat.com>
Patchwork-id: 102152
O-Subject: [RHEL7 drm 11/89] drm/radeon: re-enable dpm by default on BTC
Bugzilla: 1173317
RH-Acked-by: Jerome Glisse <jglisse@redhat.com>
RH-Acked-by: Dave Airlie <airlied@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1173317
Upstream: since v3.16.4

commit b47f2102b4862f881c25b3c5e14f179e54413089

Author:     Alex Deucher <alexander.deucher@amd.com>
AuthorDate: Mon Jul 14 12:01:40 2014 -0400
Commit:     Rob Clark <rclark@redhat.com>
CommitDate: Wed Dec 10 17:08:39 2014 -0500

    drm/radeon: re-enable dpm by default on BTC

    commit c08abf11900e19b14dd3a0cc3d105bd74519cd18 upstream.

    This patch depends on:
    e07929810f0a19ddd756558290c7d72827cbfcd9
    (drm/radeon/dpm: fix typo in vddci setup for eg/btc)

    bugs:
    https://bugs.freedesktop.org/show_bug.cgi?id=73053
    https://bugzilla.kernel.org/show_bug.cgi?id=68571

    Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Rob Clark <rclark@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/gpu/drm/radeon/radeon_pm.c b/drivers/gpu/drm/radeon/radeon_pm.c
index a0f3cec..a18596e 100644
--- a/drivers/gpu/drm/radeon/radeon_pm.c
+++ b/drivers/gpu/drm/radeon/radeon_pm.c
@@ -1316,9 +1316,6 @@ int radeon_pm_init(struct radeon_device *rdev)
  case CHIP_RS780:
  case CHIP_RS880:
  case CHIP_RV770:
- case CHIP_BARTS:
- case CHIP_TURKS:
- case CHIP_CAICOS:
   /* DPM requires the RLC, RV770+ dGPU requires SMC */
   if (!rdev->rlc_fw)
    rdev->pm.pm_method = PM_METHOD_PROFILE;
@@ -1342,6 +1339,9 @@ int radeon_pm_init(struct radeon_device *rdev)
  case CHIP_PALM:
  case CHIP_SUMO:
  case CHIP_SUMO2:
+ case CHIP_BARTS:
+ case CHIP_TURKS:
+ case CHIP_CAICOS:
  case CHIP_CAYMAN:
  case CHIP_ARUBA:
  case CHIP_TAHITI:
-- 
1.7.1